DESCRIPTION

```diff
@@ -10,6 +10,10 @@ Authors@R:
              email = "andrea.vicini@eurac.edu",
              role = "aut",
              comment = c(ORCID = "0000-0001-9438-6909")),
+      person(given = "Roger", family = "Gine Bertomeu", 
+             email = "roger.gine@iispv.cat",
+             comment = c(ORCID = "0000-0003-0288-9619"),
+             role = "aut"),
```

Comment:
Please add below in `Imports` `(>= 1.11.2) for `MsCoreUtils` to ensure that the correct version of the package providing this function is used.